### PR TITLE
STORM-2244 Some shaded jars doesn't exclude dependency signature files

### PR DESCRIPTION
**Pull request by HeartSaVioR**
* Add filter option to all of shade plugin configurations

### CODE REVIEW

File: examples/storm-elasticsearch-examples/pom.xml [hunk 1]
```diff
@@ -62,6 +62,23 @@
                 <artifactId>maven-shade-plugin</artifactId>
                 <configuration>
                     <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <filters>
```
Comment (srdo) [inline examples/storm-elasticsearch-examples/pom.xml]: I'm wondering if it would work to put this in PluginManagement in the root pom instead?

**Comment by harshach** [comment]
+1

**Comment by hmcl** [comment]
+1

**Comment by harshach** [comment]
@HeartSaVioR can you open a PR for 1.x-branch as well.

**Comment by HeartSaVioR** [comment]
@harshach Just missed this. I'll create pull request against 1.x branch soon.

**Pull request merged**